Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/shotgun.yml

```diff
@@ -1,13 +1,12 @@
 ﻿- type: entity
   id: ShellShotgunBase
   name: shell (.50)
-  parent: BaseItem
+  parent: BaseItemCartridge
   abstract: true
   components:
   - type: Tag
     tags:
     - ShotgunShell
-    - Cartridge
```

Comment:
Tags don't merge with parent, so this is needed still.

Reply:
Thought this might be the case, thanks. Luckily I've got a new idea for this PR so happy it didn't get merged.